src/libraries/Common/src/SourceGenerators/ImmutableEquatableArray.cs

```diff
@@ -72,15 +73,12 @@ public bool MoveNext()
         }
     }
 
-    public static class ImmutableEquatableArray
+    internal static class ImmutableEquatableArray
```

Comment:
Why mark the helpers internal when the type itself is public?

Reply:
The helpers are currently only used internally.